app/controllers/v0/institutions_controller.rb

```diff
@@ -88,5 +88,14 @@ def facets
       }
     end
     # rubocop:enable AbcSize
+
+    # Embed search result counts as a list of hashes with "name"/"count"
+    # keys so that open-ended strings such as country names do not
+    # get interpreted/mutated as JSON keys.
+    def embed(group_counts)
+      group_counts.each_with_object([]) do |(k, v), array|
+        array << { name: k, count: v }
+      end
```

Comment:
This is already done with a database group/count query by means of the filter_counts method in the facets method above. This new method is just re-structuring the result of that query.